### PR TITLE
Fixed #29823 -- Doc'd limitation of DecimalField on SQLite.

### PR DESCRIPTION
**Pull request by claudep**

### CODE REVIEW

File: docs/ref/databases.txt [hunk 1]
```diff
@@ -677,6 +677,19 @@ substring filtering.
...
+.. _sqlite-decimal-handling:
+
+Decimal handling
+----------------
+
+SQLite has no real decimal internal type. Decimal values are internally
+converted to the ``REAL`` data type (8-byte IEEE floating point number), as
+explained in the `SQLite datatypes documentation`__. So if you care about the
+precision of decimal values stored in your database, you should avoid using
+SQLite.
```
Comment (felixxm) [inline docs/ref/databases.txt]: I think the last sentence is a bit harsh :smile:, maybe:
```
explained in the `SQLite datatypes documentation`__, so they don't support
correctly-rounded decimal floating point arithmetic.

__ https://www.sqlite.org/datatype3.html#storage_classes_and_datatypes
```

Comment (claudep) [reply docs/ref/databases.txt]: Suggestion accepted!

**Comment by felixxm** [comment]
buildbot

**Comment by felixxm** [comment]
@claudep Thanks!

**Comment by felixxm** [comment]
buildbot

**Pull request merged**